pgmpy/tests/test_readwrite/test_XMLBIF.py

```diff
@@ -2,7 +2,17 @@
 import unittest
 import numpy as np
 import numpy.testing as np_test
-from pgmpy.readwrite import XMLBIFReader
+from pgmpy.readwrite import XMLBIFReader, XMLBIFWriter
+try:
+    from lxml import etree
+except ImportError:
+    try:
+        import xml.etree.cElementTree as etree
+    except ImportError:
+        try:
+            import xml.etree.ElementTree as etree
+        except ImportError:
+            warnings.warn("Failed to import ElementTree from any known place")
```

Comment:
Okay. But do create an issue before the PR is merged, so we know that this needs to be simplified.